js/apps/admin-ui/src/authentication/RequiredActions.tsx

```diff
@@ -65,6 +89,8 @@ export const RequiredActions = () => {
     try {
       if (field in action) {
         action[field] = !action[field];
+        // remove configurable property from action which only exists for the admin ui
+        delete action.configurable;
```

Comment:
We need to omit the `configurable` property here. This is necessary since we use the configurable action representation in the admin UI with the `configurable` property, which currently does not exist on the required action representation used by the standard admin rest API.